### ISSUE TITLE
Feature Request: Documentation in PDF format

### ISSUE THREAD

**Issue by cluettig** [hubject/oicp#20]
It would be good to have generated PDF of the current OICP Documentation in order exchange it locally within a Product or Development Team.

This would be beneficial for exchanging information's within the Team or also across Teams in one Company, througthComments in the PDF File.

It should be fairly easy to create it based on the ASCIIDoc files and could be possibel to generate automatically on creating a new Release within Github

**Comment by carstenhag** [comment]
Agree, I thought I was able to access the PDF at the releases, but those weren't there.

**Comment by carstenhag** [comment]
Stumbled across this again, 4.5 years later! Oh well.
